src/main/index.js

```diff
@@ -75,6 +75,23 @@ const makeFullUrl = (url, search = null) =>
         }
     ));
 
+/**
+ * Prompt in a platform-specific way for permission to access the microphone or camera, if Electron supports doing so.
+ * Any application-level checks, such as whether or not a particular frame or document should be allowed to ask,
+ * should be done before calling this function.
+ *
+ * @param {string} mediaType - one of Electron's media types, like 'microphone' or 'camera'
+ * @returns {boolean} - true if permission granted, false otherwise.
+ */
+const askForMediaAccess = async mediaType => {
+    if (systemPreferences.askForMediaAccess) {
+        // Electron currently only implements this on macOS
+        return await systemPreferences.askForMediaAccess(mediaType);
```

Comment:
nit: `return await` [is unnecessary](https://eslint.org/docs/rules/no-return-await)
```suggestion
        return systemPreferences.askForMediaAccess(mediaType);
```

Reply:
Yeah, I felt a bit weird writing that but I figured it was convention. Looks like I was wrong -- thanks for letting me know about that eslint rule! We should probably add it to `eslint-config-scratch`...